api/job/janitor_test.go

```diff
@@ -3,17 +3,43 @@ package job
 import (
 	"testing"
 	"time"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestJanitor(t *testing.T) {
 	store := NewMemoryStore()
 	janitor := NewJanitor(store, time.Hour*24)
```

Comment:
you should probably make `expiry` a variable, and set the `Created` field below based off of those times, e.g. 
```
// delete
Created: now.Sub(expiry+1),
...
// keep
Created: now.Sub(expiry-1),
```

Reply:
Sub uses time not duration. https://golang.org/pkg/time/#Time.Sub

Reply:
huh weird that `Add` and `Sub` wouldn't be the opposite of eachother 🤷‍♂️